### PR TITLE
Environment configuration for HTTPS only cookie

### PR DESCRIPTION
**Pull request by jasonmccreary**
As this likely changes between environments, figured it should be part of the environment configuration. Not entirely happy with the name though. While descriptive, `SESSION_HTTPS_ONLY_COOKIES` seemed long.

### CODE REVIEW

File: config/session.php [hunk 1]
```diff
@@ -161,7 +161,7 @@
     |
     */
 
-    'secure' => false,
+    'secure' => env('SESSION_SECURE_COOKIE', false),
```
Comment (it-can) [inline config/session.php]: I think ```SESSION_COOKIE_SECURE``` is a better name

Comment (GrahamCampbell) [inline config/session.php]: Why not `SECURE_SESSIONS`?

Comment (jasonmccreary) [inline config/session.php]: Was trying to match the other names with the `SESSION_` prefix. If you're open to breaking trend, I like `SECURE_SESSION_COOKIES` best.

Comment (jeroennoten) [inline config/session.php]: I think `SESSION_SECURE` is the right choice, because we already have `SESSION_DOMAIN`.

**Pull request merged**